src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -69,6 +70,9 @@ def generate_info(revision, coveralls_token):
       '-t', 'coveralls',
       '-s', REPO_DIR,
       '-p', '/home/worker/workspace/build/src/',
+      '--service-name', 'TaskCluster',
+      '--service-number', datetime.today().strftime('%Y%m%d'),
+      '--service-job-number', '1',
```

Comment:
In the future, service-job-number will define the test suite that generated the code coverage information. So we will upload multiple artifacts, each one with a different service-job-number, corresponding to each test suite (mochitest, xpcshell, etc.).